### DISCUSSION TITLE
安装IDE以后，应该做什么。

### DISCUSSION THREAD

**Discussion by breakToT** [ByteLegend/ByteLegend#53] [Q&A]
我完成了酒吧里安装JAVA的任务，勇士榜的星星，安装IDE的星星。现在我需要做什么呢，酒吧叫我清理幺蛾子，也找不到哪个地方能去了啊。新手村就这么大，点完了都。

**Comment by gtn1024** [comment]
后面还没开发完，慢慢等

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: breakToT ***@***.***>
Sent: Tuesday, September 14, 2021 1:35:11 PM
To: ByteLegend/ByteLegend ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ByteLegend/ByteLegend] 安装IDE以后，应该做什么。 (Discussion #53)


我完成了酒吧里安装JAVA的任务，勇士榜的星星，安装IDE的星星。现在我需要做什么呢，酒吧叫我清理幺蛾子，也找不到哪个地方能去了啊。新手村就这么大，点完了都。

―

**Reply by breakToT** [reply]
我以为是我还没有找到正确的打开的点。
